ale/rotation.py

```diff
@@ -179,6 +179,10 @@ def __mul__(self, other):
         if isinstance(other, ConstantRotation):
             return TimeDependentRotation((self._rots * other._rot).as_quat(), self.times, other.source, self.dest)
         elif isinstance(other, TimeDependentRotation):
+            # if self and other each have the same time and one rotation, don't interpolate.
+            if (self.times.size == 1) and (other.times.size == 1):
+                if (self.times == other.times):
+                    return TimeDependentRotation((self._rots * other._rots).as_quat(), self.times, other.source, other.dest)
```

Comment:
This should be `other.source, self.dest)`. If other rotates a to b and this rotates b to c. Then, self * other rotates  a to c.

Reply:
Thank you for catching this. Totally a typo.